### ISSUE TITLE
In the latest version of webui, it is not working properly

### ISSUE THREAD

**Issue by chenxluo** [gitmylo/sd-webui-riffusion-outpaint#7]
Actually, it has not been working since webui version after March 29th. The output result is only an image with a mask, and the error message is as follows:

'int' object is not iterable
'int' object is not iterable

**Comment by gitmylo** [comment]
On what lines? i haven't really been updating the repo for a bit, and use vladmandic's fork, there might be differences.
and i haven't tested it recently either.

**Comment by chenxluo** [comment]
> On what lines? i haven't really been updating the repo for a bit, and use vladmandic's fork, there might be differences. and i haven't tested it recently either.

I am using the a1111 webui master branch. It has been updated with the gradio library after March 29th, which caused many of the previous plugins to not work properly.

**Comment by gitmylo** [comment]
Right, but what line is the error on? this error is probably not gradio related

**Comment by chenxluo** [comment]
Sorry, I am not very familiar with webui plugins. I can only see this error message in the console: ‘int’ object is not iterable. When I roll back to a version March 29th, the plugin works properly again. So I guess this is a critical version

**Comment by smonzon14** [comment]
Hi there, I'm a little late to the party but I can confirm this error on current webui even today. It's because the img2img function signature has changed again and the parameters are all swapped around. There are quite a few params out of place that I'm trying to fix but not sure if there is an alternative out there?

    Traceback (most recent call last):
      File "C:\Users\smonz\Downloads\sd.webui\webui\modules\scripts.py", line 872, in postprocess
        script.postprocess(p, processed, *script_args)
      File "C:\Users\smonz\Downloads\sd.webui\webui\extensions\sd-webui-riffusion-outpaint\scripts\riffusion_outpaint.py", line 104, in postprocess
        (next_chunk, total) = generate_next_chunk(keep_debug, inpainting_fill_mode, length, expand_amount,
      File "C:\Users\smonz\Downloads\sd.webui\webui\extensions\sd-webui-riffusion-outpaint\scripts\riffusion_outpaint.py", line 178, in generate_next_chunk
        generate_with_inpaint_source = generate_img2img(inpaint_source, inpaint_mask, inpainting_fill_mode,
      File "C:\Users\smonz\Downloads\sd.webui\webui\extensions\sd-webui-riffusion-outpaint\scripts\riffusion_outpaint.py", line 205, in generate_img2img
        return img2img.img2img(
      File "C:\Users\smonz\Downloads\sd.webui\webui\modules\img2img.py", line 153, in img2img
        override_settings = create_override_settings_dict(override_settings_texts)
      File "C:\Users\smonz\Downloads\sd.webui\webui\modules\infotext_utils.py", line 417, in create_override_settings_dict
        for pair in text_pairs:
    TypeError: 'int' object is not iterable

**Comment by gitmylo** [comment]
The issue is most likely caused by an update from a while ago. Since stable diffusion webui's code changes, sometimes there could be breaking changes, especially when using internal apis from stable diffusion webui. As this repo hasn't been updated in 2 years, and I haven't used stable diffusion webui in nearly 2 years either, (as I've switched to comfyui and swarmui), it isn't going to get fixed unless someone volunteers and writes a pull request.

I don't expect the ui to have changed that much in the used sections though.

In short, the issue is caused by the code being outdated, but I will not be updating it myself.
